### PR TITLE
Header other pages

### PR DESCRIPTION
**Pull request by NataliaLKB**
Updating Header for non home pages to include the logo, which goes to the home page

Here is how it looks:

image
- IMPORTANT \* Please merge #27 first, as I didn't want merge conflicts :blush:

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -64,82 +64,116 @@ footer.page-footer {
   NAV BAR
 ***************************/
 
-.nav-list {
+.nav-list--home {
```
Comment (NataliaLKB) [inline css/style.css:67]: Updating styling to be more "BEM" like

File: events.html [hunk 2]
```diff
@@ -22,11 +22,12 @@
 <body>
   <nav class="white" role="navigation">
     <div class="nav-wrapper container">
-      <ul class="hide-on-med-and-down nav-list">
-        <li><a class="nav-list__list-item" href="./team.html">Meet the Team</a></li>
-        <li><a class="nav-list__list-item" href="./events.html">Upcoming Events</a></li>
-        <li><a class="nav-list__list-item" href="#">Resources</a></li>
-        <li><a class="nav-list__list-item" href="#">Contact Us</a></li>
+      <ul class="hide-on-med-and-down nav-list nav-list--with-image">
+        <li class="nav-list__list-item"><a href="./team.html">Meet the Team</a></li>
```
Comment (NataliaLKB) [inline events.html:26]: Also moved the class onto the `li` to better reflect the name (list-item)

File: css/style.css (hunk 1 above)
Comment (iteles) [inline css/style.css:67]: :heart:

**Comment by iteles** [comment]
Loving the BEM modifier syntax!

I've opened issues https://github.com/node-girls/node-girls-website/issues/33 but don't think it's a blocker for this PR (or even for the first iteration of the website) so merging this in :+1:

**Pull request merged**